src/components/Shared/FormShell.js

```diff
@@ -357,6 +361,38 @@ export default function FormShell({
                 })
               }
               break
+            case 'onClearGridConfirmation':
+              action.onClick = () => {
+                stack({
+                  Component: ClearGridPhyConfirmation,
+                  props: {
+                    open: [true, {}],
+                    fullScreen: false,
+                    onConfirm: action.onSuccess
+                  },
+                  width: 450,
+                  height: 170,
+                  expandable: false,
+                  title: platformLabels.Clear
+                })
+              }
+              break
+            case 'onClearAllConfirmation':
+              action.onClick = () => {
+                stack({
+                  Component: ClearFilteringPhy,
+                  props: {
+                    open: [true, {}],
+                    fullScreen: false,
+                    onConfirm: () => handleReset()
+                  },
+                  width: 450,
+                  height: 170,
+                  expandable: false,
+                  title: platformLabels.Clear
+                })
+              }
+              break
```

Comment:
these two functions are not needed here you can send them directly from the actions array (you can check the Close button in TransactionForm file)

Reply:
you liked this comment, but there is no change regarding it why?